version-check/action.yml

```diff
@@ -0,0 +1,32 @@
+name: "Version Check"
+description: "Checks to make sure the version matches the correct format"
+inputs:
+  version:
+    description: 'Version string to check'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Check Runner OS
+      shell: bash
+      env:
+        RUNNER_OS: ${{ runner.os }}
+      run: |
+        if [ "$RUNNER_OS" != "Linux" ]; then
+          echo "[!] This workflow only supports Linux runners"
+          exit 1
+        fi
+
+    - name: Check version format
+      id: check-version-format
+      shell: bash
```

Comment:
❓  Even with `shell: bash` this doesn't support Windows/Mac?

Reply:
Looks like this does run on all runners.  I was under the impression composite actions had a sort of limitation to Linux. https://github.com/bitwarden/server/actions/runs/8173747940